core/app/modules/site-editor/assets/scss/admin/admin-bar.scss

```diff
@@ -1,4 +1,4 @@
-#wpadminbar #wp-admin-bar-elementor_app_site_editor a.ab-item {
+#wpadminbar #wp-admin-bar-elementor_edit_page a.elementor-general-section.first {
```

Comment:
Use a pseudo selector 

```suggestion
#wpadminbar #wp-admin-bar-elementor_edit_page a.elementor-general-section{
```

Reply:
Not possible, @StyleShit please suggest alternative.

Reply:
You can use the `+` selector:
```HTML
<ul>
    <li class="a"></li>
    <li class="a"></li>
    <li class="b"></li>
    <li class="b"></li>
</ul>
```

```CSS
.a + .b {
    // whatever
}
```

Reply:
Thanks.

Reply:
Handled.